corpus/gutenberg/README.md

```diff
@@ -2,18 +2,42 @@
 
 A few books can be taken from this repository. So far, we have picked:
 
-- Alice’s Adventures in Wonderland (Lewis Carroll) —
-  [de](https://www.gutenberg.org/ebooks/19778),
-  [en](https://www.gutenberg.org/ebooks/11),
-  [fr](https://www.gutenberg.org/ebooks/55456),
-  [it](https://www.gutenberg.org/ebooks/28371)
-- Don Quixote (Miguel de Cervantes Saavedra) —
-  [en](https://www.gutenberg.org/ebooks/996),
-  [es](https://www.gutenberg.org/ebooks/2000),
-  [fr](https://www.gutenberg.org/ebooks/42524)
+| Book                                               | de       | en       | es       | fr       | it       |
+| :------------------------------------------------- | :------: | :------: | :------: | :------: | :------: | 
+| Alice’s Adventures in Wonderland (Lewis Carroll)   | [de][1]  | [en][2]  |          | [fr][3]  | [it][4]  |
+| Don Quixote (Miguel de Cervantes Saavedra)         |          | [en][5]  | [es][6]  | [fr][7]  |          |
+| The Adventures of Pinocchio (Carlo Collodi)        |          | [en][8]  |          |          | [it][9]  |
+| The Count of Monte Cristo (A. Dumas and A. Maquet) |          | [en][10] |          | [fr][11], [fr][12], [fr][13], [fr][14] |   |
+| Faust (Johann Wolfgang von Goethe)                 | [de][15], [de][16] | [en][17] | [es][18] | [fr][19] |   |
```

Comment:
Suggestion: sort the lines so that the writer’s native language matches the order of the columns (de, en, es, fr, it).